CHANGELOG.md

```diff
@@ -4,6 +4,51 @@
 
 ### Notable changes
 
+* **V8**: upgrade to 4.2.77.18, see the [ChangeLog](https://chromium.googlesource.com/v8/v8/+/refs/heads/4.2.77/ChangeLog) for full details. Notable items:
+  - Classes have moved out of staging, the `class` keyword is now usable without flags
```

Comment:
Did we get those free when they enabled classes such that we can use them in POJSO's too? `{ foo() { return 'bar' } }`? Would you mind having a quick go at some words for this for me?